app/index.html

```diff
@@ -59,15 +59,15 @@
         </script>
 
         <script type="text/html" id="template-add-tree">
-            <label><input type="radio" name="type" value="tree">Tree</label>
-            <label><input type="radio" name="type" value="bush">Bush</label>
-            <label><input type="radio" name="type" value="herb">Herb</label>
-            <br>
-            <br>
-            <br>
-            <input id="description" type="text" name="description" placeholder="Description">
-            <br>
-            <br>
+            <div class="form-button">
+                <input id="radio1" type="radio" name="type" value="tree" checked>
+                <label for="radio1">Tree</label>
+                <input id="radio2" type="radio" name="type" value="bush">
+                <label for="radio2">Bush</label>
+                <input id="radio3" type="radio" name="type" value="herb">
+                <label for="radio3">Herb</label>
+                <input id="description" type="text" name="description" placeholder="Description">
```

Comment:
I think the description field should be on its own line (block level) and be a `<textarea>` (multi-line text) field instead of `<input>`.